frontend/public/module/k8s/get-resources.ts

```diff
@@ -99,6 +107,15 @@ export const pluralizeKind = (kind: string): string => {
 export const getResources = () =>
   coFetchJSON('api/kubernetes/apis').then((res) => {
     const preferredVersions = res.groups.map((group) => group.preferredVersion);
+    const groupVersionMap = fromJS(
+      res.groups.reduce((acc, { name, versions, preferredVersion: { version } }) => {
+        acc[name] = {
+          versions: _.map(versions, 'version'),
+          preferredVersion: version,
```

Comment:
We're already storing preferred versions in Redux. We should only put it in one place, whether it's in the map or the original value.

Reply:
The preferred version that is already present maps from `group~Version` to preferred version.

Reply:
Removing the older one replacing the ones that use it.